db/migrate/20200128111859_populate_user_dishes_count.rb

```diff
@@ -0,0 +1,7 @@
+class PopulateUserDishesCount < ActiveRecord::Migration[5.0]
+  def up
+    Dish.all.each do |dish|
+      Dish.reset_counters(dish.id, :user_dishes)
+    end
+  end
```

Comment:
Isn't there a way to reset counters for all the Dishes at once?

Reply:
AFAIK there isn't any built in method to do so in one request